### DISCUSSION TITLE
Working with ptz presets

### DISCUSSION THREAD

**Discussion by artorias994** [roleoroleo/yi-hack-Allwinner#364] [Q&A]
Hi to all!

I'm having troubles working with ptz presets, from UI i see the 'go to preset' option, but I can't understand how do I save those presets, i.e. Move manually the camera to x position and save position to preset `#1`

Can someone give me some hints?

Thank you!

**Comment by roleoroleo** [comment]
At the moment you can set presets only with the app.
And after you set them, you can move with the web page or the web service.
I'm working on a new version that is able to set them form the web page but it's work in progress.
